src/libsync/networkjobs.h

```diff
@@ -75,11 +75,11 @@ class OWNCLOUDSYNC_EXPORT LsColXMLParser : public QObject
     void finishedWithoutError();
 };
 
-class OWNCLOUDSYNC_EXPORT LsColJob : public AbstractNetworkJob
+class OWNCLOUDSYNC_EXPORT PropfindJob : public AbstractNetworkJob
 {
     Q_OBJECT
 public:
-    explicit LsColJob(AccountPtr account, const QUrl &url, const QString &path, QObject *parent = nullptr);
+    explicit PropfindJob(AccountPtr account, const QUrl &url, const QString &path, int depth = 1, QObject *parent = nullptr);
```

Comment:
Since we allow only two values, it might be easier (and safer) to introduce one or two named constructors.